Examples/Io/Performance/ActsExamples/Io/Performance/CKFPerformanceWriter.hpp

```diff
@@ -66,6 +66,9 @@ class CKFPerformanceWriter final : public WriterT<TrajectoriesContainer> {
     DuplicationPlotTool::Config duplicationPlotToolConfig;
     TrackSummaryPlotTool::Config trackSummaryPlotToolConfig;
 
+    /// Whether to do double matching or not
+    bool doubleMatching = false;
```

Comment:
does it make sense to make this default on?

Reply:
I didn't want to break any hashes... do we have this in our physmon?
In principle I would make this the default...

Reply:
yes I think this would change a bunch of reference files